app/src/main/java/org/simple/clinic/patient/Languages.kt

```diff
@@ -0,0 +1,9 @@
+package org.simple.clinic.patient
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class Languages(
```

Comment:
This class is no longer required now, right?

Reply:
Oh right, I forgot to delete this.